server/channels/wsapi/system.go

```diff
@@ -20,3 +22,18 @@ func ping(req *model.WebSocketRequest) (map[string]any, *model.AppError) {
 
 	return data, nil
 }
+
+func (api *API) websocketNotificationAck(req *model.WebSocketRequest) (map[string]any, *model.AppError) {
+	// Log the ACKs if necessary
+	api.App.NotificationsLog().Debug("Websocket notification acknowledgment",
```

Comment:
I think I somehow misunderstood you. From what I understood, we were deferring the logging aspect and decided that we would use a custom logging level for this, and for the immediate scope we would only be adding a new metric. But I see that you are using the notificationsLog for this. Perhaps there was a miscommunication?

Reply:
Is this absolutely necessary for this PR to go in? I know we want to have these custom log levels, and I do have a ticket to do this: https://mattermost.atlassian.net/browse/MM-57027

But I don't think we need to change this right here, right now. I'd like to do all the logging changes all in one shot. This is just one small log to add to the already existing notifications log and I think it's fine to leave as-is.

Reply:
Ok, then I misunderstood you.